File: app/src/main/java/com/spidchenko/week2task/models/ImgSearchResult.java
package com.spidchenko.week2task.models;

import com.google.gson.annotations.SerializedName;

public class ImgSearchResult {

    @SerializedName("photos")
    private ImageContainer mImageContainer;

    @SerializedName("stat")
    private String mStat;

    public ImageContainer getImageContainer() {
        return mImageContainer;
    }
}

File: app/src/main/java/com/spidchenko/week2task/network/models/ImgSearchResult.kt
package com.spidchenko.week2task.network.models;

import com.google.gson.annotations.SerializedName;

public class ImgSearchResult {

    @SerializedName("photos")
    private ImageContainer mImageContainer;

    public ImageContainer getImageContainer() {
        return mImageContainer;
    }
}
